ansible/inventory

```diff
@@ -138,6 +143,8 @@ hkgmetadb.infra.ooni.io
 datacollector.infra.ooni.io
 hkgmetadb.infra.ooni.io
 amsmetadb.ooni.nu
+# db-1.proteus.test.ooni.io
+# db-1.proteus.ooni.io
```

Comment:
I added `log_connections` so I plan to roll out firewall on these hosts tomorrow.
@hellais tell me, if that's a bad idea.